frontend/src/lib/lemon-ui/LemonInput/LemonInput.scss

```
@@ -85,4 +89,10 @@
         width: 100%;
         max-width: 100%;
     }
+
+    .LemonInputSelect & {
+        flex-wrap: wrap;
+        height: auto;
+        min-height: var(--lemon-input-height); // TODO: Does this break anything?
```

Looks ok based on snapshots
```suggestion
        min-height: var(--lemon-input-height);
```